osgi.specs/docbook/159/service.feature.xml

```diff
@@ -729,6 +707,35 @@ Feature f = builder.build();</programlisting>
         
     </section>
   </section>
+
+
+  <section>
+    <title>Framework Properties</title>
```

Comment:
This should be called Framework Launching Properties. See <https://docs.osgi.org/specification/osgi.core/8.0.0/framework.lifecycle.html#framework.lifecycle.launchingproperties>.